pytorch_lightning/trainer/model_hooks.py

```diff
@@ -1,53 +0,0 @@
-# Copyright The PyTorch Lightning team.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-from abc import ABC
-from typing import Optional
-
-import pytorch_lightning as pl
-from pytorch_lightning.utilities import rank_zero_deprecation
-from pytorch_lightning.utilities.signature_utils import is_param_in_hook_signature
-
-
-class TrainerModelHooksMixin(ABC):
```

Comment:
This PR should update the CHANGELOG, although it's not available yet. You'll have to wait until https://github.com/PyTorchLightning/pytorch-lightning/pull/10316 is merged.